### ISSUE TITLE
1.19 Support

### ISSUE THREAD

**Issue by BobtheBob123456** [Camotoy/GeyserSkinManager#20]
Hi,
With the new Wild Update just having come out, we am wondering when/if this plugin is updating to support 1.19.

Can we please have an approximate date?

Thanks

**Comment by Camotoy** [comment]
Does it not work out of the box in 1.19 somewhere?

**Comment by BobtheBob123456** [comment]
I don't know if it already works on 1.19. On the spigot page it just says it supports 1.16 to 1.18 so assumed it didn't, but it might. I just haven't tested it

**Comment by BobtheBob123456** [comment]
Wait, so does the plugin work on 1.19 or not?

**Comment by BobtheBob123456** [comment]
So does the plugin work for 1.19? If so, could the spigot page be updated to say it.

**Comment by BobtheBob123456** [comment]
Wait, so does the plugin work for 1.19? If so, could the spigot page be updated to say it.

**Comment by Camotoy** [comment]
Yes, the current release should have 1.19 support. I would encourage you to try the plugin next time instead of waiting for a response. But, my apologies for the delay. The Spigot page will be updated when I get around to making a new release.
